package.json

```diff
@@ -10,8 +10,8 @@
   "homepage": "https://github.com/sibiraj-s/action-eslint#readme",
   "main": "lib/main.js",
   "engines": {
-    "node": ">=16.0.0",
-    "npm": ">=8.0.0"
+    "node": ">=20.0.0",
+    "npm": ">=10.0.0"
```

Comment:
> "npm": ">=10.0.0"

Took a look at https://nodejs.org/en/about/previous-releases and saw that npm v10 is listed alongside the latest v20 release, so I assumed that v10 is the minimum that goes with Node v20.

Reply:
Thanks, Mostly this pinning won't be an issue, it just for the workspace or when published to npm. but its good to get the right version set to avoid confusions.